app/Jobs/ResetEnvironment.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+namespace App\Jobs;
+
+use App\Enums\Logs\JobType;
+use App\Events\Jobs\EnvironmentResetCompleted;
+use Exception;
+use Illuminate\Bus\Queueable;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Foundation\Bus\Dispatchable;
+use Illuminate\Queue\InteractsWithQueue;
+use Illuminate\Queue\SerializesModels;
+use Illuminate\Support\Facades\Artisan;
+use Illuminate\Support\Facades\Redis;
+
+class ResetEnvironment implements ShouldQueue
+{
+    use Dispatchable;
+    use InteractsWithQueue;
+    use Queueable;
+    use SerializesModels;
+
+    protected $commands = [
+        [
+            'command' => 'migrate:fresh ',
+        ],
+        [
+            'command' => 'app:init-permissions',
+        ],
+        [
+            'command' => 'db:seed',
+        ],
+        [
+            'command' => 'view:clear',
+        ],
+        [
+            'command' => 'route:clear',
+        ],
+        [
+            'command' => 'config:clear',
+        ],
+        [
+            'command' => 'config:cache',
+        ],
+        [
+            'command' => 'clear-compiled',
+        ],
+        [
+            'command' => 'app:update-users',
+        ],
+        [
+            'command' => 'app:update-websites',
+        ],
+    ];
+
+    public function handle(): void
+    {
```

Comment:
Qui dovrebbe essere previsto un controllo che impedisca di attivare il job in ambiente di produzione.